test/unit/org/apache/cassandra/cql3/statements/DescribeStatementTest.java

```diff
@@ -1004,7 +1004,9 @@ private static String tableParametersCql()
                "    AND memtable_flush_period_in_ms = 0\n" +
                "    AND min_index_interval = 128\n" +
                "    AND read_repair = 'BLOCKING'\n" +
-               "    AND speculative_retry = '99p';";
+               "    AND speculative_retry = '99p'\n" +
+               "    AND automated_repair_full = {'enabled': 'true'}\n" +
```

Comment:
np: should we maybe nix the "automated" prefix?

Reply:
Incorporated